index.html

```diff
@@ -1,11 +1,757 @@
 <!DOCTYPE html>
-  <html>
-      <head>
-        <title>NY Times Clone</title>
-        <link href="styles.css" rel="stylesheet" type="text/css">
-        <meta charset="utf-8">
-      </head>
-      <body>
-        
-      </body>
-  </html>
\ No newline at end of file
+<html>
+  <head>
+    <title>NY Times Clone</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <header>
+          <div class="box-icons">
+            <div class="header-icon">
+              <img
+                src="https://img.icons8.com/ios-filled/50/000000/menu.png"
+                alt="hamburger icon">
+            </div>
+            <div class="header-icon">
+              <img
+                src="https://img.icons8.com/material/24/000000/zoom-out.png"
+                alt="search">
+            </div>
+            <div>
+              <span class="spc font">SPACE & COSMOS</span>
```

Comment:
`REQUIRED`

This should be a link as in the original page. See below:

![Screenshot of nav links](https://res.cloudinary.com/bolaah/image/upload/v1563048062/github-microverse-project/code-reviews/003.gif)